frappe/public/js/frappe/form/controls/link.js

```diff
@@ -467,10 +466,16 @@ frappe.ui.form.ControlLink = frappe.ui.form.ControlData.extend({
 		for(var i=0; i < fl.length; i++) {
 			frappe.model.set_value(df.parent, docname, fl[i], fetch_values[i], df.fieldtype);
 		}
+	},
+
+	filter_awesomplete: function(awesomplete) {
+		awesomplete.filter = function() {
+			return true;
+		};
 	}
 });
```

Comment:
Do we need this? Can't we just set `awesomplete.filter` in `table_multiselect.js`?

Reply:
Awesome filter was already set in link fields so dint want to change the functionality